eagle-jpm/eagle-jpm-spark-history/src/main/resources/application.conf

```diff
@@ -45,7 +48,6 @@
     }
   },
   "storm":{
-    worker.num: 2,
```

Comment:
Since the app framwork has set the worker number and the this conf has been removed, we need to also move the conf in SparkHistoryJobAppProvider.xml?